ChangeLog.d/fix-pkcs12-null-password.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix
+   * Fix issues in pkcs12 when a NULL and/or zero length password was supplied.
```

Comment:
Changelog entries should be more specific than “issues”. Also they should mention the GitHub issue when there is one. Suggested wording:

> Fix a potential invalid pointer dereference in pkcs12 functions when the password is empty. Fix the documentation which incorrectly described the password as optional and potentially null. Fixes #5136.